cliff.toml

```diff
@@ -3,7 +3,9 @@
 
 [changelog]
 # template for the changelog footer
-header = """---"""
+header = """
+
+"""
```

Comment:
would this not break the PR description editing?

Reply:
Right. Then I should post-process the `---` in the `docs.md` files of the mender-docs-changelog, because they are not needed there.